cfme/tests/control/test_bugs.py

```diff
@@ -7,12 +7,13 @@
 from widgetastic.widget import Text
 
 from cfme import test_requirements
+from cfme.control.explorer import conditions
 from cfme.control.explorer import ControlExplorerView
 from cfme.control.explorer.alert_profiles import ServerAlertProfile
-from cfme.control.explorer.conditions import VMCondition
 from cfme.control.explorer.policies import VMCompliancePolicy
 from cfme.control.explorer.policies import VMControlPolicy
 from cfme.exceptions import CFMEExceptionOccured
+from cfme.tests.control.test_basic import CONDITIONS
```

Comment:
Could you move such constants used in many test suits to some common place ?
cross references between test suits are frowned upon.
I belive you can move them to
```
cfme/tests/control/__init__.py
or 
cfme/control/__init__py
```

Reply:
ignore above